### DISCUSSION TITLE
Does mysql client supports connection pooling?

### DISCUSSION THREAD

**Discussion by vikramcse** [PyMySQL/mysqlclient#707] [Q&A]
Does mysql client supports connection pooling in Django?

the django docs mentions that the mysqlclient driver supports pooling

![image](https://github.com/PyMySQL/mysqlclient/assets/1611331/69d4a09e-6d82-4155-849e-597fffe13203)

**Comment by methane** [comment]
This driver doesn't provide pool.
Django ORM and SQLAlchemy does.
